.travis.yml

```diff
@@ -100,4 +100,4 @@ notifications:
   email:
     on_success: never
     on_failure: always
-  webhooks: https://pulls.web-platform-tests.org/api/build
+  webhooks: https://pulls-staging.web-platform-tests.org/api/build
```

Comment:
Are you sure this should be in this PR?

Reply:
-  webhooks: https://pulls.web-platform-tests.org/api/build
    +  webhooks: https://pulls-staging.web-platform-tests.org/api/build
> Are you sure this should be in this PR?

@lukebjerring Is there a reason you went ahead and merged the branch for this change to master without responding to this review comment?

Reply:
No - accidentally done due to lack of familiarity with github, and unfortunately the comment wasn't flagged as needing action. Sorry for any inconvenience - I quickly reverted immediately afterward.